03/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="description" content="Letra de la canción 'Vete de una vez' de Daniela Spalla">
+  <meta name="keywords" content="daniela, spalla, vete, vez, cancion">
+  <title>Vete de una vez - Daniela Spalla</title>
+</head>
+<body>
+  <main>
+    <header>
+      <h1>Vete de una vez</h1>
+      <h2>Daniela Spalla</h2>
+    </header>
+  </main>
+    <section>
+      <h4>Intro</h4>
+    </section>
+    <section>
+      <h5>He pasado tantos días, tantas noches
+        Haciendo el duelo, el duelo por ti
+        Lo dijiste claramente no querías
+        Una vida junto a mí
+        Pero cada tanto vuelves, me llamas
+        Y sabes que jamás voy a decir que no
+        Cada tanto vuelves
+        Basta, te pido por favor</h5>
+    </section>
+    <aside>
+      <h4>Chorus</h4>
+    </aside>
+    <section>
+      <h5>
+        Vete de una vez y ya no vuelvas
+        Que mi corazón está cansado y ya no puede más
+        Ya perdí todo el honor, llegas y matas lo poco
+        De cordura que me queda y no me dejas avanzar
+        Vete de una vez, que me haces daño
+        ¿No ves que mi alma tiene un límite para el dolor?
+        Yo te di todo mi amor, no lo quisiste y ahora
+        Debes aprender a estar sin mí
+      </h5>
+    </section>
+    <article>
+      <h4>Outro</h4>
+    </article>
+    <section>
+      <h5>
+        He llorado tantos días, tantas noches
+        Haciendo el duelo, el duelo por ti
+        Lo dijiste claramente no querías
+        No podías imaginarte un futuro junto a mí
+        Pero cada tanto vuelves, me llamas
+        Y sabes que jamás voy a decir que no
+        Si, cada tanto vuelves
+        Basta, te pido por favor
+      </h5>
+    </section>
+    <section>
+      <h4>Chorus</h4>
+      <h5>
+        Vete de una vez y ya no vuelvas
+        Que mi corazón está cansado y ya no puede más
+        Ya perdí todo el honor, llegas y matas lo poco
+        De cordura que me queda y no me dejas avanzar
+        Vete de una vez, que me haces daño
+        ¿No ves que mi alma tiene un límite para el dolor?
+        Yo te di todo mi amor, no lo quisiste y ahora
+        Debes aprender a estar sin mí
+      </h5>
+    </section>
+    <div></div>
+    <footer>
```

Comment:
Why main is not the main tag in this document?
If you have more doubts, take a look at this link https://dev.to/sapardo/etiquetas-semanticas-de-html5-3j7m